tests/integration/test_package_managers.py

```diff
@@ -90,6 +90,19 @@
             ),
             id="gomod_vendor_check_correct_vendor",
         ),
+        # Test case checks if package can be replaced with local dependency
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachito-gomod-local-deps.git",
+                ref="b2e465b91a6a272540c77d4dde1e317773ed700b",
+                packages=({"path": ".", "type": "gomod"},),
+                flags=["--gomod-vendor-check"],
```

Comment:
is `--gomod-vendor-check` an intended part of the test?

Reply:
Nice catch! This was an accidental copy ..